### ISSUE TITLE
How connect your software to a ip socket? 

### ISSUE THREAD

**Issue by Wooopz** [Lecerof/TelemetryViewer#13]
Hello, how can i do to connect the software to a ip socket instead of serial port ? Making a linux special device maybe ? 
Thank you

**Comment by Lecerof** [comment]
I am not sure, but I am rather busy with other projects at the moment so this one is on ice until the summer unfortunately. But do you mean that you want to log data from something connected through the ethernet?

**Comment by Wooopz** [comment]
Yes, a high altitude balloon with a raspberry PI zero, GPS, and sensors through 3G connection like this project : http://zedomax.com/blog/2010/03/17/diy-arduino-based-balloon-gps-tracking-device/

**Comment by Wooopz** [comment]
After much research, I was able to connect the telemetry viewer via a wifi connection using the Windows "Virtual Serial Port" driver and the "Esp Link" firmware in esp8266.
You will find useful information here:
https://caron.ws/diy-cartes-microcontroleurs/raspberrypi/connexion-esp8266/
